test/distributed/test_store.py

```diff
@@ -259,15 +258,18 @@ def _multi_worker_helper(self, world_size):
         server_store = create_tcp_store(addr, world_size, wait_for_workers=False)
         server_store.set("key", "value")
         port = server_store.port
-        world_size = random.randint(5, 10) if world_size == -1 else world_size
-        for i in range(world_size):
-            self._create_client(i, addr, port, world_size)
+
+        if world_size and world_size > 0:
+            for i in range(world_size):
+                self._create_client(i, addr, port, world_size)
+        else:
+            self._create_client(0, addr, port, None)
```

Comment:
nit: this could be simplified by keeping for loop and doing something like:

```python
num_indices = world_size if world_size else 1
for i in range(num_indices):
   self._create_client(i, addr, port, world_size)
```